lib/tapjoy/autoscaling_bootstrap.rb

```diff
@@ -154,13 +154,22 @@ def confirm_config(keypair:, zones:, security_groups:, instance_type:,
       end
 
       # configure environment
-      def configure_environment(filename, env=nil, config_dir)
-        defaults_hash = self.load_yaml("#{config_dir}/config/common/defaults.yaml")
-        facet_hash    = self.load_yaml("#{config_dir}/config/clusters/#{filename}")
+
+      def configure_environment(filename, env=nil, config_dir=nil)
+        if filename.include?(File::SEPARATOR)
+          facet_file    = filename
+          config_dir   = "#{File.expand_path('../../..', facet_file)}"
+        else
+          facet_file    = "#{config_dir}/config/clusters/#{filename}"
+        end
+
+        common_path   = "#{config_dir}/config/common/"
+        defaults_hash = self.load_yaml("#{common_path}/defaults.yaml")
```

Comment:
A more idiomatic way to build file paths would be to use File.join("part1","part2","part3"), which would result in "part1/part2/part3" (the first part needs a leading slash if you intend for it to lead with one, which File.expand_path does).

That's not really a blocking concern, but just something to keep in mind